src/flowcean/polars/transforms/discrete_derivative.py

```diff
@@ -0,0 +1,116 @@
+import logging
+from collections.abc import Iterable
+from typing import Literal, TypeAlias
+
+import polars as pl
+from typing_extensions import override
+
+from flowcean.core import Transform
+
+logger = logging.getLogger(__name__)
+
+DiscreteDerivativeType: TypeAlias = Literal["forward", "backward", "central"]
```

Comment:
```suggestion
DiscreteDerivativeKind: TypeAlias = Literal["forward", "backward", "central"]
```
I don't know what the convention is in Python, but I feel like this is not a _type_